samples/kubeflow-tf/kubeflow-training-classification.py

```diff
@@ -51,6 +51,11 @@ def kubeflow_tf_training_op(transformed_data_dir, schema: 'GcsUri[text/json]', l
         ],
         file_outputs = {'train': '/output.txt'}
     )
+    if use_gpu:
+        kubeflow_tf_training_op.image = 'gcr.io/ml-pipeline/ml-pipeline-kubeflow-tf-trainer-gpu:85c6413a2e13da4b8f198aeac1abc2f3a74fe789',
```

Comment:
`.image` needs to be an string, not tuple.

Reply:
nice catch